diff --git a/Template/LO/SubProcesses/survey.sh b/Template/LO/SubProcesses/survey.sh
index ebe9369..9617b7e 100755
--- a/Template/LO/SubProcesses/survey.sh
+++ b/Template/LO/SubProcesses/survey.sh
@@ -12,6 +12,10 @@ if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRT" ]; then
   fi
 fi
 
+if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRTDEL" ]; then
+  export LD_LIBRARY_PATH="$SRT_LD_LIBRARY_PATH_SCRAMRTDEL:$LD_LIBRARY_PATH"
+fi
+
 # If TMPDIR is unset, set it to the condor scratch area if present
 # and fallback to /tmp
 export TMPDIR=${TMPDIR:-${_CONDOR_SCRATCH_DIR:-/tmp}}
diff --git a/Template/NLO/SubProcesses/ajob_template b/Template/NLO/SubProcesses/ajob_template
index 6144c59..64dcca5 100755
--- a/Template/NLO/SubProcesses/ajob_template
+++ b/Template/NLO/SubProcesses/ajob_template
@@ -19,6 +19,10 @@ if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRT" ]; then
   fi
 fi
 
+if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRTDEL" ]; then
+  export LD_LIBRARY_PATH="$SRT_LD_LIBRARY_PATH_SCRAMRTDEL:$LD_LIBRARY_PATH"
+fi
+
 # If TMPDIR is unset, set it to the condor scratch area if present
 # and fallback to /tmp
 export TMPDIR=${TMPDIR:-${_CONDOR_SCRATCH_DIR:-/tmp}}
diff --git a/Template/LO/SubProcesses/refine.sh b/Template/LO/SubProcesses/refine.sh
index 87cc4a8..8ce1d3d 100644
--- a/Template/LO/SubProcesses/refine.sh
+++ b/Template/LO/SubProcesses/refine.sh
@@ -12,6 +12,10 @@ if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRT" ]; then
   fi
 fi
 
+if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRTDEL" ]; then
+  export LD_LIBRARY_PATH="$SRT_LD_LIBRARY_PATH_SCRAMRTDEL:$LD_LIBRARY_PATH"
+fi
+
 # If TMPDIR is unset, set it to the condor scratch area if present
 # and fallback to /tmp
 export TMPDIR=${TMPDIR:-${_CONDOR_SCRATCH_DIR:-/tmp}}
diff --git a/Template/LO/SubProcesses/refine_splitted.sh b/Template/LO/SubProcesses/refine_splitted.sh
index bd1b78c..827dfc3 100644
--- a/Template/LO/SubProcesses/refine_splitted.sh
+++ b/Template/LO/SubProcesses/refine_splitted.sh
@@ -12,6 +12,10 @@ if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRT" ]; then
   fi
 fi
 
+if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRTDEL" ]; then
+  export LD_LIBRARY_PATH="$SRT_LD_LIBRARY_PATH_SCRAMRTDEL:$LD_LIBRARY_PATH"
+fi
+
 # If TMPDIR is unset, set it to the condor scratch area if present
 # and fallback to /tmp
 export TMPDIR=${TMPDIR:-${_CONDOR_SCRATCH_DIR:-/tmp}}
diff --git a/Template/NLO/SubProcesses/reweight_xsec_events.local b/Template/NLO/SubProcesses/reweight_xsec_events.local
index fa64a85..491445b 100644
--- a/Template/NLO/SubProcesses/reweight_xsec_events.local
+++ b/Template/NLO/SubProcesses/reweight_xsec_events.local
@@ -15,6 +15,10 @@ if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRT" ]; then
   fi
 fi
 
+if [ -n "$SRT_LD_LIBRARY_PATH_SCRAMRTDEL" ]; then
+  export LD_LIBRARY_PATH="$SRT_LD_LIBRARY_PATH_SCRAMRTDEL:$LD_LIBRARY_PATH"
+fi
+
 # If TMPDIR is unset, set it to the condor scratch area if present
 # and fallback to /tmp
 export TMPDIR=${TMPDIR:-${_CONDOR_SCRATCH_DIR:-/tmp}}
